install/install.md

```diff
@@ -147,7 +210,7 @@ docker run -t -d --gpus all --shm-size=1g --ulimit memlock=-1 -v $PWD:/ws <conta
 
```

Comment:
Does the `--ulimit` change here need to be updated to include the same magic number as in the release selector?

Reply:
@ajschmidt8 when you have a sec - does this docker command need to be updated to match the one in the release selector? Specifically thinking about `--ulimit stack`

Reply:
Did a lot of digging, it looks like most Linux OSes default to a `ulimit stack` of 8MB vs the 64MB we specify here. Without this specification, we can run into crashes because 8MB isn't enough for our stack.

@exactlyallan can you update the command in this line from
> docker run -t -d --gpus all --shm-size=1g --ulimit memlock=-1 -v $PWD:/ws

to

> docker run -t -d --gpus all --shm-size=1g --ulimit memlock=-1  --ulimit stack= 67108864 -v $PWD:/ws